types/moji/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "no-irregular-whitespace": false
```

Comment:
For new package types, it should not include other rules

Reply:
moji-tests.ts
```
moji('　').convert('ZS', 'HS').toString();
```
This test code causes an error.

Reply:
You can change to below code to just disable the rule:
```
// tslint:disable-next-line:no-irregular-whitespace
moji('　').convert('ZS', 'HS').toString();
```

Reply:
You really saved me.